### ISSUE TITLE
whiledo/ilias-transientmysql does not work anymore

### ISSUE THREAD

**Issue by tikiatua** [kekru/ilias-docker#1]
Hi there,

Thank you for providing the docker files for ilias. We are currently evaluation ilias and were using the image whiledo/ilias-transientmysql:latest to setup the system. 

However, it seems that the container is somewhat broken, as the internal mysql instance will not start anymore.

Here is the log from the container:
```
 * Stopping MySQL database server mysqld
   ...done.
 * Starting MySQL database server mysqld
   ...fail!
mysqladmin: connect to server at '127.0.0.1' failed
error: 'Can't connect to MySQL server on '127.0.0.1' (111)'
Check that mysqld is running on 127.0.0.1 and that the port is 3306.
You can check this by doing 'telnet 127.0.0.1 3306'
running: 127.0.0.1
ERROR 2003 (HY000): Can't connect to MySQL server on '127.0.0.1' (111)
```

We are starting the container without any special configuration using docker-compose:
```
ilias:
    container_name: ilias
    image: whiledo/ilias-transientmysql:latest
```

**Comment by kekru** [comment]
Thank you for creating the issue. 
I created a fix on #2 :)

**Comment by tikiatua** [comment]
Hi @kekru,

Thank you very much for fixing this so quickly. Works great.
